docs/en/trust.rst

```diff
@@ -602,28 +604,83 @@ The Trust Chains can also be verified offline, using one of the Trust Anchor's p
+In its Entity Configuration the Relying Party publishes its specific metadata, including the supported signature and encryption algorithms
+and any other necessary information for the interoperability requirements.
 
-Any requests for User attributes, such as PID or (Q)EAA, from the Relying Party to Wallet Instances are signed and SHOULD contain the verifiable Trust Chain regarding the Relying Party.
+Any requests for User attributes, such as PID or (Q)EAA, from the Relying Party to Wallet Instances are signed and SHOULD contain the verifiable
+Trust Chain regarding the Relying Party.
 
-The Wallet Instance verifies that the Trust Chain related to the Relying Party is still active, proving that the Relying Party is still part of the Federation and not revoked.
+The Wallet Instance verifies that the Trust Chain related to the Relying Party is still active,
+proving that the Relying Party is still part of the Federation and not revoked.
 
-The Trust Chain SHOULD be contained within the signed request in the form of a JWS header parameter.
+The Trust Chain MAY be contained within the signed request in the form of a JWS header parameter.
 
 In offline flows, Trust Chain verification enables the assessment of the reliability of Trust Marks and Attestations contained within.
 
 
+Establishing Trust with Credential Issuers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In the issuance process, trust evaluation ensures the integrity and authenticity of the Credentials being issued and the realiability of their Issuers. This section delineates the trust evaluation mechanisms distinct from the protocol flows, implemented by Wallet Instances and Relying Parties, as described in the dedicated section.
+
+Trust evaluations implement different ways, as defined below:
+
+* **Federation Entity Discovery**: Wallet Instances and Relying Parties MUST verify the identity of the Issuer through a Federation Entity Discovery process. This involves querying a trusted list or directory to confirm the Issuer's validity status and compliance with the Trust Framework.
+
+* **Trust Chains**: Wallet Instances and Relying Parties evaluate Issuer's Trust Chains, be provided statically or build though a Federation Entity Discovery process, to ensure that the entity requesting the Credential is part of a recognized and trusted federation. This involves checking the Trust Chain from the root authority to the Issuer.
+
+* **Trust Marks Evaluation**: Trust Marks are assessed to ensure ongoing compliance with federation policies. These marks indicate adherence to specific standards and practices required by the federation.
+
+* **Policy Evaluation**: Wallet Instances and Relying Parties MUST check that the Credential Issuer is allowed in the issuance of the Credential of their interest. Metadata, metadata policies and Trust Marks are used for the implementation of these checks.
```

Comment:
Same comment as for the trust marks applies here.

Reply:
as above